cmd/aks-periscope/aks-periscope.go

```diff
@@ -14,83 +15,91 @@ import (
 
 func main() {
 	zipAndExportMode := true
-	exporter := &exporter.AzureBlobExporter{}
+	exporters := []interfaces.Exporter{}
+	clusterType := os.Getenv("CLUSTER_TYPE")
+
+	exporters = append(exporters, &exporter.AzureBlobExporter{})
```

Comment:
If this is always appended just init wiht it already there, i.e. `[]interfaces.Exporter{&exporter.AzureBlobExporter{}}`